src/routes/middlewares/schema-validator.middleware.js

```diff
@@ -0,0 +1,23 @@
+import Joi from 'joi';
+
+import { errorResponse } from '../../utils/helpers.utils';
+/**
+ * @description joi validation schema
+ * @param {object} schema
+ * @returns {object} {req, res, next}
+ */
+export default function validateReqBody(schema) {
```

Comment:
This has already been implemented, please rebase to get the latest from develop.

Reply:
Thank, for that.
I have rebased and found an existing method that handles that.